src/plugins/intel_gpu/src/graph/include/program_node.h

```diff
@@ -191,6 +191,9 @@ struct program_node {
     layout calc_output_layout() const;
     std::vector<layout> calc_output_layouts() const;
 
+    kernel_impl_params get_fake_aligned_params(kernel_impl_params const& orig_impl_param) {
+        return std::move(orig_impl_param);
+    }
```

Comment:
is it used?

Reply:
It might be used in the future if it is to be used in static shape, but not now. So I will remove it. Thanks!